# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/mips32/Assembler-mips32.h
# Commit: 1b94af3458ce
# Full Hash: 1b94af3458ce8bf47b776e03ac74f0c1b5a72c65
# Author: Dmitry Bezhetskov <dbezhetskov@igalia.com>
# Date: 2020-09-11 15:10:42
# Regressor Bug: 1639153
# File Overlap Count: 1
# Description:
#   Bug 1639153 - Part 1: Reserve two slots after stack arguments for the future tls preservation. r=lth
#   
#   
#   We are going to remove Frame::tls and support trampolines for indirect calls, so we need to get rid of using Frame::tls.
#   In this and the followup patches I will iteratively remove all dependencies of Frame::tls and remove it eventually.
# ==============================================================================

diff -r e127d8c36c87 -r 1b94af3458ce js/src/jit/mips32/Assembler-mips32.h
--- a/js/src/jit/mips32/Assembler-mips32.h	Fri Sep 11 11:52:12 2020 +0300
+++ b/js/src/jit/mips32/Assembler-mips32.h	Thu Aug 27 08:56:29 2020 +0000
@@ -43,6 +43,8 @@
 
     return usedArgSlots_ * sizeof(intptr_t);
   }
+
+  void increaseStackOffset(uint32_t bytes) { MOZ_CRASH("NYI"); }
 };
 
 // These registers may be volatile or nonvolatile.